### PR TITLE
Fix/crisp support

### PR DESCRIPTION
**Pull request by guru-web3**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

File: src/pages/ProviderChange.vue [hunk 1]
```diff
@@ -48,18 +50,25 @@ const finalProviderData = reactive<FinalTxData>({
...
+      finalProviderData.origin = providerData.origin;
+      finalProviderData.toNetwork = providerData.newNetwork.displayName;
+      finalProviderData.fromNetwork = providerData.currentNetwork;
+      finalProviderData.whitelabelData = providerData.whitelabelData;
+    } else {
+      finalProviderData.toNetwork = params.displayName;
+      finalProviderData.fromNetwork = ControllerModule.torus.currentNetworkName;
+    }
+  } catch (error) {
+    log.error(error, "error in tx");
```
Comment (metallicalfa2) [inline src/pages/ProviderChange.vue:70]: Throw the error after handle

File: src/App.vue [hunk 2]
```diff
@@ -4,13 +4,14 @@ import { onBeforeMount } from "vue";
...
 import ControllerModule from "./modules/controllers";
-import { hideCrispButton, isMain } from "./utils/helpers";
+import { hideCrispButton, isIframe, isMain } from "./utils/helpers";
 
 onBeforeMount(() => {
   if (isMain) ControllerModule.init({ origin: window.location.origin });
 
   // hide crispbutton for iframe
-  if (!isMain) {
+  if (!isIframe) {
```
Comment (ieow) [inline src/App.vue:13]: why !isIFrame ?
this made crisp always show on iframe

File: src/pages/ProviderChange.vue [hunk 3]
```diff
@@ -48,18 +50,27 @@ const finalProviderData = reactive<FinalTxData>({
...
+      const providerData = await bcHandler.getMessageFromChannel<ProviderChangeChannelEventData>();
+      finalProviderData.origin = providerData.origin;
+      finalProviderData.toNetwork = providerData.newNetwork.displayName;
+      finalProviderData.fromNetwork = providerData.currentNetwork;
+      finalProviderData.whitelabelData = providerData.whitelabelData;
+    } else {
+      finalProviderData.toNetwork = params.displayName;
+      finalProviderData.fromNetwork = ControllerModule.torus.currentNetworkName;
+    }
+    throw new Error("test");
```
Comment (metallicalfa2) [inline src/pages/ProviderChange.vue:69]: remove this

**Review by metallicalfa2 — APPROVED** [review]
lgtm

**Pull request merged**